.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -8,70 +8,70 @@ jobs:
     vmImage: ubuntu-latest
   strategy:
     matrix:
-      linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.10.____cpythonpython_implcpython:
-        CONFIG: linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.10.____cpythonpython_implcpython
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.0
-      linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.7.____cpythonpython_implcpython:
-        CONFIG: linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.7.____cpythonpython_implcpython
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.0
-      linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.8.____cpythonpython_implcpython:
-        CONFIG: linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.8.____cpythonpython_implcpython
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.0
-      linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.9.____cpythonpython_implcpython:
-        CONFIG: linux_64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9python3.9.____cpythonpython_implcpython
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.0
-      linux_64_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9python3.10.____cpythonpython_implcpython:
-        CONFIG: linux_64_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9python3.10.____cpythonpython_implcpython
+      ? linux_64_c_compiler_version10cuda_compiler_version11.1cxx_compiler_version10fortran_compiler_version10python3.10.____cpythonpython_implcpython
+      : CONFIG: linux_64_c_compiler_version10cuda_compiler_version11.1cxx_compiler_version10fortran_compiler_version10python3.10.____cpythonpython_implcpython
```

Comment:
This is bizarre. Idk why it is prefixing the matrix builds this way

Reply:
This is valid yaml for long keys.

Reply:
I thought I configured the bot to remove this but I think it still somehow snuck in.

Reply:
Thanks for clarifying. TIL

Ok sounds like it was still unexpected though?

Reply:
Yes it is.

Reply:
Ohhhh nvm. Conda smithy wrote that. It is fine.